### DISCUSSION TITLE
How to turn off automatic recalculation?

### DISCUSSION THREAD

**Discussion by meine-dicke** [osmandapp/OsmAnd#18636] [Q&A]
Hi, I´m a newbie using OSMAnd+ and coming from Germany.

When I have imported a route (ex from basecamp) to OSMAnd+ and follow this route, the route will be automatic recalculated if I leave this route.
If I don´t want this recalculation of the route, because I just want to go few km/miles next to the route and then follow the (old) route again:

    Is that even possible? and if yes
    How can I turn off the automatic recalculation?

Thank you for helping
meine-dicke

**Comment by DmitryAlexei** [comment]
Please see: 
- https://osmand.net/docs/user/navigation/setup/gpx-navigation (navigation by GPX)
- https://osmand.net/docs/user/navigation/guidance/navigation-settings#recalculate-route (route recalculation)

**Reply by meine-dicke** [reply]
Thank you very much for the links.
